### ISSUE TITLE
Questions about model training

### ISSUE THREAD

**Issue by jonDel** [RasaHQ/rasa_nlu#2391]
**Rasa Core version**:
rasa-core           0.9.8

**Python version**: 
Python 3.6.6 :: Anaconda, Inc.

**Operating system** (windows, osx, ...):
Fedora Linux

**Issue**:
Considering rasa default deep learning model, what is the size/proportion to training data of:
- validation set:
- test set?
Is there an early stopping strategy, or the training session just goes as long as the pre-defined number of epochs (no specific treatment to prevent overfitting)?

**Comment by mbelang** [comment]
@michaelcapizzi We did exactly what you suggest and it doesn't work at all. I tried to play with the parameters of the early stopping callback to make it stop very soon and we set a default to 200 epochs with a very small set of data. I see the `val_acc` is constant for more than the patience allow but it doesn't stop..
